dapps/marketplace/src/pages/user/User.js

```diff
@@ -17,6 +17,11 @@ import { getProviderDisplayName } from 'utils/profileTools'
 const User = ({ match }) => {
   const id = match.params.id
   const vars = { id: match.params.id }
+
+  if (!id) {
```

Comment:
Instead of this we can just add `skip={!id}` to the `Query` component do the query doesn't even get run if id isn't present

Reply:
is that ideal?  I'd kind of like to know if this is an issue.  Though I did take one other shot in the dark at fixing this as well.

Reply:
I've been doing it elsewhere in the code and think it's fine, yea